core-skills/tdd/README.md

```diff
@@ -24,91 +24,104 @@
 
-- [ ] Can explain when you should and shouldn’t use test doubles.
+- Can explain when you should and shouldn’t use test doubles.
   - Required - one example of when a test double is useful.
   - Required - one example of when a test double is NOT useful.
 
-- [ ] Can explain why minimizing the number of test doubles a single test case uses is desirable
+- Can explain why minimizing the number of test doubles a single test case uses is desirable
   - Required - a sensible example of how having too many test doubles can cause problems/problematic tests.
 
-- [ ] Can explain benefits of the different ways of testing remote services.
+- Can explain benefits of the different ways of testing remote services.
   - Required - Can identify three methods of remote testing.
   - Required - Can explain the pros and cons of each.
 
-- [ ] Can implement an appropriate method of testing a remote service.
+- Can implement an appropriate method of testing a remote service.
   - Required - Passing test code which handles a remote service adequately.
 
-- [ ] Can describe the situations where using triangulation helps improve test quality.
+- **DEPRECATED, BUT MUST BE ACHIEVED IF NOT DONE IN GIRAFFE** - Can describe the situations where using triangulation helps improve test quality.
   - Required - Can explain what the potential flaws of not including triangulation are.
   - Required - Can explain how triangulation overcomes these flaws.
 
-- [ ] Can demonstrate triangulation when TDDing an example piece of code.
+- **DEPRECATED, BUT MUST BE ACHIEVED IF NOT DONE IN GIRAFFE** - Can demonstrate triangulation when TDDing an example piece of code.
   - Required - Production code cannot be easily slimed, due to the use of triangulation.
 
 ### Dragon
 
 ![Dragon](./dragon.jpg)
 
-Refactor dem code.
-
-- [ ] Does not TDD markup.
-  - Required - No example of this, unless explained with good reason
-
-- [ ] Can identify smells within code they have written
-
-- [ ] Can identify smells within somebody else's code
+Review and refactor a preprepared test suite to improve performance.
 
-- [ ] Can explain the balancing act of achieving conciseness of a test suite
+- Can write an acceptance test around an existing feature.
+  - Required - Write an acceptance test around a feature of an existing application which verifies the feature works as intended.
 
-- [ ] Can describe why test suite performance is important
+- Can identify smells within somebody else's code.
+  - Required - Can list three valid smells per the invigilator's guide.
 
-- [ ] Can identify structural defects that result in poor test suite performance
+- Can describe why test suite performance is important.
+  - Required - See invigilator's guide.
 
-- [ ] Can identify that a test suite has poor test suite performance
+- Can identify causes of poor performance in a test suite.
+  - Required - Identify three of the secret causes of poor performance.
```

Comment:
three secret causes :O

Reply:
Better than "knows the secret smells"

Reply:
I'm not a big fan of the secrecy thing. I feel it's complicated and brittle.

We felt there were particular bits of knowledge that _must_ be known by the candidate, and that putting them in the marking guide wouldn't test that.